network-libp2p/src/network.rs

```diff
@@ -177,15 +179,28 @@ impl PubsubId<PeerId> for GossipsubId<PeerId> {
     }
 }
 pub struct Network {
+    /// The local ID that is used to identify our peer
     local_peer_id: PeerId,
-    connected_peers: Arc<RwLock<HashSet<PeerId>>>,
+    /// This hash map mantains an association between PeerIds and PeerContact:
+    /// If the peer is interesting, i.e.: it provides services that are interested to us,
+    ///  we store a some value with the peer contact itself
+    /// Otherwise,
+    ///  we just store a None value
+    ///
```

Comment:
```suggestion
    /// we store a some value with the peer contact itself
    /// Otherwise,
    /// we just store a None value
```
(extra spaces and lines)